networking_oneview/ml2/drivers/oneview/synchronization.py

```diff
@@ -57,9 +57,14 @@ def get_session(self):
     def synchronize(self):
         self.delete_outdated_flat_mapped_networks()
         self.create_oneview_networks_from_neutron()
-        self.delete_unmapped_oneview_networks()
-        self.synchronize_uplinkset_from_mapped_networks()
-        self.recreate_connection()
+
+        force_delete = common.CONF.oneview.force_sync_delete_ops
+
+        LOG.debug("'force_sync_delete_ops' is set to", force_delete)
```

Comment:
please, rewrite this message for something more readable for the cloud administrator, variable name it's not a useful information. And change to LOG.info.